frontend/simple/sass/bulma_overrides/elements/form.scss

```diff
@@ -5,3 +5,18 @@
     color: rgba($input-color, 0.4); // To improve A11Y
   }
 }
+
+.gi- {
+  &is-grouped-reverse {
+    flex-direction: row-reverse;
+
+    &.field.is-grouped-right {
+      justify-content: flex-start;
+    }
+
+    &.field.is-grouped > .control:not(:last-child) {
+      margin-left: 0.75rem;
```

Comment:
Cool! I would just replace this `0.75rem` for the correct variable to avoid hardcoded code. `$control-padding-horizontal` following Bulma docs.

Reply:
Indeed, as this could turn out to be something that gets adjusted in themes as well.

Reply:
I actually copied this from the bulma source: https://github.com/jgthms/bulma/blob/master/sass/elements/form.sass#L479 and just switched the direction. so in non-gi-specific grouped fields, that padding will always be 0.75rem and won't be adjusted by any variable. 

I agree, It would totally make more sense to use a variable here, but then it wouldn't be consistent with other similar bulma classes.

so, should I still proceed and change this to a var?